app/models/claim/reconciled_claim.rb

```
@@ -0,0 +1,38 @@
+      :billing_provider_type_code,
+      :claim_coverage_type,
+      :claim_information_source,
+      :date_of_birth,
+      :first_date_of_service,
+      :full_name,
+      :gender,
+      :procedure_code
+    )
+
+    def client_id
+      return unless ite_client_id
```

rubocop (1.69.1)

[convention] Redundant `return` detected.

(Style/RedundantReturn)